docs/data/destinations/moengage-event-streaming.md

```diff
@@ -46,13 +46,18 @@ Create an access token in Extole. You need this to complete the Amplitude setup.
 ![a diagram that shows Moengage setup steps](/assets/images/moengage-api-key.png)
 6. Paste your **Basic Auth Token**, **DATA APP ID** and **Data Center** information.
     1. Data Center Values: “01”, “02”, “03”, “04”. You can refer to this [article](https://help.moengage.com/hc/en-us/articles/360057030512-Data-Centers-in-MoEngage) for more details on identifying your MoEngage data center.
-7. Toggle the Send events filter to select the events to send. You can send all events, but Amplitude recommends choosing the most important ones.
-8. Use the Event Properties filter to select which Event Properties you would like to send.
-9. When finished, save your work.
+7. Toggle the **Send events** filter to select the events to send. You can send all events, but Amplitude recommends choosing the most important ones.
+8. Use the **Event Properties** filter to select which Event Properties you would like to send.
+9. Under **Send Users**, make sure the toggle is enabled ("Users are sent to MoEngage") if you want to stream users and their properties to MoEngage. When enabled, users are automatically created or updated in MoEngage when an event is sent to Amplitude.
+10. When finished, save your work.
 
+## View forwarded events from Amplitude in MoEngage
+1. Log into MoEngage.
+2. Click on **Events** tab.
```

Comment:
```suggestion
2. Navigate to the the **Events** tab.
```

Reply:
Ooops I should probarbly change my template to make sure I incorporate these changes. Thank you!